docs/topics/clusterdefinitions.md

```diff
@@ -486,6 +455,20 @@ Below is a list of kube-scheduler options that are _not_ currently user-configur
 
 We consider `kubeletConfig`, `controllerManagerConfig`, `apiServerConfig`, and `schedulerConfig` to be generic conveniences that add power/flexibility to cluster deployments. Their usage comes with no operational guarantees! They are manual tuning features that enable low-level configuration of a kubernetes cluster.
 
+#### Arbitary YAML for Kubernetes component manifests
```

Comment:
Slight preference for "Custom" over "Arbitrary". To me "arbitrary" has a negative connotation and implies randomness.